packages/grafbase-graphiql/src/utils/sse.tsx

```diff
@@ -84,25 +82,27 @@ export const SSEProvider = ({ children }: { children?: ReactNode }) => {
    */
   const sseClient = (url: string) => {
     setStatus('CONNECTING')
```

Comment:
Would be nice to have them as an enum so the user can SSEStatus.Connecting etc... instead of plain strings with type.

Reply:
Sure, but I need a better status subscription
Ideally, `sseClient` should be extracted to sse lib and reused – setStatus gets in the way

Reply:
Understood!